go.mod

```diff
@@ -37,7 +37,7 @@ require (
 	github.com/go-openapi/validate v0.19.5 // indirect
 	github.com/go-redis/redis v6.15.5+incompatible
 	github.com/gogo/googleapis v1.3.0 // indirect
-	github.com/gomodule/redigo v1.7.0
+	github.com/gomodule/redigo v2.0.0+incompatible
```

Comment:
I do not think this is needed in this PR.

Reply:
no, but not sure where this came from actually. will revert it.